stable/sonarqube/values.yaml

```diff
@@ -42,10 +54,19 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: "-"
+  storageClass: ""
```

Comment:
Why this change?

Reply:
My bad. restored the storageClass: "-" value.